### DISCUSSION TITLE
[false positive] VirusTotal Analyse

### DISCUSSION THREAD

**Discussion by alex-messer** [mifi/lossless-cut#1559] [Support]
Hello all,
I just downloaded the latest version of your project.

Before unpacking and using it, I threw the download link into VirusTotal to see if there was anything wrong with your project / code....

And unfortunately VirusTotal found something...

Security vendors' analysis
| Name | Type | Company | Status |
| ------ |------ |------ |------ |
| Lumu | Malware | Abusix | Clean |

The analysis shows the status "Clean", but it still makes me think...

Could you tell me something about this?

https://github.com/mifi/lossless-cut/releases/download/v3.54.0/LosslessCut-win-x64.7z

https://www.virustotal.com/gui/url/5af94b993b25d6f1f8bd408a9c6741c68d5f9e1535f8912fef610cb0b1b37e38/detection

**Comment by mifi** [comment]
Hi. I don’t know what that means. Where can I get more info about this «Lumu» malware?

**Reply by photonometric** [reply]
TL;DR - The 1 positive from this report is from a _URL scanner for phishing websites_, which returned a **false positive for the GitHub site** itself. See my longer responses below for more detailed explanations about how VirusTotal's reports sometimes include scans of the source website, as well as the file itself. 

### **All scans of the LosslessCut release files were 100% clean**.

**Comment by photonometric** [comment]
[Lumu](https://lumu.io/) isn't the malware, it's one of what VT calls [contributors](https://support.virustotal.com/hc/en-us/articles/115002146809-Contributors) or "vendors," which is a distinct threat detection database (many of these are just retail Antivirus programs that VT aggregates.) Also note that the long column of 'vendors' wraps around to a 2nd column set; _Abusix_ is the name of another detection entity (despite being named like the evil supercomputer overlord in an awful 80s action movie), and that one returned a clean result. 

Apparently Lumu maintains some kind of enterprise IT-oriented threat database. According to the vendor link above, in VT scanning it is only used for URL and domain scans? Which I don't know much about (maybe phishing etc?)—and that it sees no problem reporting false positives, which VT subsequently includes as part of the full (i.e. explanationless) scan score. 

And while it's possible to use a free API key to retrieve slightly more informational portions of AV file scanning (see next post)...most of the API keywords for getting any useful info about why the GitHub wite seemed suspicious there, is only available to the top tier of enterprise subscribers. 

Oh well, carry on! (I bought TMPEG Smart Renderer last year, works great, but I'm very sad it discards any subtitles instead of cutting them. Very eager to see your new smart render and subtitle capabilities! :)

**Comment by photonometric** [comment]
OK! Not that exciting because the actual file scan AV detection was 0, but at least we can see how?

This was fairly easy to retrieve from the [file-info](https://developers.virustotal.com/reference/file-info) API page, just needed to type my API key and the LosslessCut release file's md5 (or an SHA) into a field each. 

<details>
  <summary>Scanner console output (file-info)</summary>
  
```
{
  "data": {
    "attributes": {
      "type_description": "7ZIP",
      "tlsh": "T11538335ED681B102DE0150BAE66E8A7C4DEE92445E1233E9B09E683F5C8E17CCD33D67",
      "type_tags": [
        "compressed",
        "7zip"
      ],
      "names": [
        "LosslessCut 3.54.0 #soft8 病毒 0 (112.04.11).7z",
        "LosslessCut-win-x64.7z",
        "losslesscut-win-x64.7z",
        "LosslessCut-win-x64 (1).7z",
        "LosslessCut-win-x64_2.7z",
        "LosslessCut-win-x64(1).7z",
        "LosslessCut-win-x64 (4).7z",
        "LosslessCut 3.54.0 Standantalone_x64.7z",
        "LosslessCut-win-x64 3.54.7z",
        "LosslessCut v3.54.0 Fr Portable.7z",
        "LosslessCut-win-x64-3.54.0.7z",
        "LosslessCut-win-x64 (April 4, 2023).7z"
      ],
      "last_modification_date": 1681183632,
      "type_tag": "7zip",
      "times_submitted": 73,
      "total_votes": {
        "harmless": 0,
        "malicious": 0
      },
      "size": 105738482,
      "type_extension": "7zip",
      "last_submission_date": 1681183632,
      "last_analysis_results": {
        "Bkav": {
          "category": "undetected",
          "engine_name": "Bkav",
          "engine_version": "1.3.0.9899",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "Lionic": {
          "category": "undetected",
          "engine_name": "Lionic",
          "engine_version": "7.5",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "Elastic": {
          "category": "type-unsupported",
          "engine_name": "Elastic",
          "engine_version": "4.0.84",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230406"
        },
        "MicroWorld-eScan": {
          "category": "undetected",
          "engine_name": "MicroWorld-eScan",
          "engine_version": "14.0.409.0",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "CMC": {
          "category": "undetected",
          "engine_name": "CMC",
          "engine_version": "2.4.2022.1",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230312"
        },
        "CAT-QuickHeal": {
          "category": "undetected",
          "engine_name": "CAT-QuickHeal",
          "engine_version": "22.00",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230409"
        },
        "ALYac": {
          "category": "undetected",
          "engine_name": "ALYac",
          "engine_version": "1.1.3.1",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "Malwarebytes": {
          "category": "undetected",
          "engine_name": "Malwarebytes",
          "engine_version": "4.5.5.54",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "Zillya": {
          "category": "undetected",
          "engine_name": "Zillya",
          "engine_version": "2.0.0.4848",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "Paloalto": {
          "category": "type-unsupported",
          "engine_name": "Paloalto",
          "engine_version": "0.9.0.1003",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "Sangfor": {
          "category": "undetected",
          "engine_name": "Sangfor",
          "engine_version": "2.23.0.0",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230404"
        },
        "K7AntiVirus": {
          "category": "undetected",
          "engine_name": "K7AntiVirus",
          "engine_version": "12.78.47679",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "Alibaba": {
          "category": "type-unsupported",
          "engine_name": "Alibaba",
          "engine_version": "0.3.0.5",
          "result": null,
          "method": "blacklist",
          "engine_update": "20190527"
        },
        "K7GW": {
          "category": "undetected",
          "engine_name": "K7GW",
          "engine_version": "12.79.47680",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "Trustlook": {
          "category": "type-unsupported",
          "engine_name": "Trustlook",
          "engine_version": "1.0",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "Arcabit": {
          "category": "undetected",
          "engine_name": "Arcabit",
          "engine_version": "2022.0.0.18",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "Baidu": {
          "category": "undetected",
          "engine_name": "Baidu",
          "engine_version": "1.0.0.2",
          "result": null,
          "method": "blacklist",
          "engine_update": "20190318"
        },
        "VirIT": {
          "category": "undetected",
          "engine_name": "VirIT",
          "engine_version": "9.5.425",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230407"
        },
        "Cyren": {
          "category": "timeout",
          "engine_name": "Cyren",
          "engine_version": "6.5.1.2",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "SymantecMobileInsight": {
          "category": "type-unsupported",
          "engine_name": "SymantecMobileInsight",
          "engine_version": "2.0",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230119"
        },
        "Symantec": {
          "category": "undetected",
          "engine_name": "Symantec",
          "engine_version": "1.19.0.0",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230409"
        },
        "tehtris": {
          "category": "type-unsupported",
          "engine_name": "tehtris",
          "engine_version": null,
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "ESET-NOD32": {
          "category": "undetected",
          "engine_name": "ESET-NOD32",
          "engine_version": "27045",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "APEX": {
          "category": "type-unsupported",
          "engine_name": "APEX",
          "engine_version": "6.406",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "TrendMicro-HouseCall": {
          "category": "timeout",
          "engine_name": "TrendMicro-HouseCall",
          "engine_version": "10.0.0.1040",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "Avast": {
          "category": "undetected",
          "engine_name": "Avast",
          "engine_version": "22.11.7701.0",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "ClamAV": {
          "category": "undetected",
          "engine_name": "ClamAV",
          "engine_version": "1.0.1.0",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "Kaspersky": {
          "category": "undetected",
          "engine_name": "Kaspersky",
          "engine_version": "22.0.1.28",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "BitDefender": {
          "category": "undetected",
          "engine_name": "BitDefender",
          "engine_version": "7.2",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "NANO-Antivirus": {
          "category": "undetected",
          "engine_name": "NANO-Antivirus",
          "engine_version": "1.0.146.25755",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "ViRobot": {
          "category": "undetected",
          "engine_name": "ViRobot",
          "engine_version": "2014.3.20.0",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "Rising": {
          "category": "undetected",
          "engine_name": "Rising",
          "engine_version": "25.0.0.27",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "Cynet": {
          "category": "type-unsupported",
          "engine_name": "Cynet",
          "engine_version": "4.0.0.27",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "Sophos": {
          "category": "undetected",
          "engine_name": "Sophos",
          "engine_version": "2.1.2.0",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "F-Secure": {
          "category": "undetected",
          "engine_name": "F-Secure",
          "engine_version": "18.10.1137.128",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "DrWeb": {
          "category": "undetected",
          "engine_name": "DrWeb",
          "engine_version": "7.0.59.12300",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "VIPRE": {
          "category": "undetected",
          "engine_name": "VIPRE",
          "engine_version": "6.0.0.35",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "TrendMicro": {
          "category": "timeout",
          "engine_name": "TrendMicro",
          "engine_version": "11.0.0.1006",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "McAfee-GW-Edition": {
          "category": "timeout",
          "engine_name": "McAfee-GW-Edition",
          "engine_version": null,
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "SentinelOne": {
          "category": "type-unsupported",
          "engine_name": "SentinelOne",
          "engine_version": "23.2.0.1",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230404"
        },
        "Trapmine": {
          "category": "type-unsupported",
          "engine_name": "Trapmine",
          "engine_version": "4.0.10.141",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230103"
        },
        "FireEye": {
          "category": "undetected",
          "engine_name": "FireEye",
          "engine_version": "35.24.1.0",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "Emsisoft": {
          "category": "undetected",
          "engine_name": "Emsisoft",
          "engine_version": "2022.6.0.32461",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "Ikarus": {
          "category": "undetected",
          "engine_name": "Ikarus",
          "engine_version": "6.1.14.0",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "Avast-Mobile": {
          "category": "type-unsupported",
          "engine_name": "Avast-Mobile",
          "engine_version": "230410-00",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "Jiangmin": {
          "category": "undetected",
          "engine_name": "Jiangmin",
          "engine_version": "16.0.100",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230409"
        },
        "Webroot": {
          "category": "type-unsupported",
          "engine_name": "Webroot",
          "engine_version": "1.0.0.403",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "Avira": {
          "category": "undetected",
          "engine_name": "Avira",
          "engine_version": "8.3.3.16",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "Antiy-AVL": {
          "category": "undetected",
          "engine_name": "Antiy-AVL",
          "engine_version": "3.0",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "Gridinsoft": {
          "category": "undetected",
          "engine_name": "Gridinsoft",
          "engine_version": "1.0.112.174",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "Xcitium": {
          "category": "undetected",
          "engine_name": "Xcitium",
          "engine_version": "35572",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "Microsoft": {
          "category": "undetected",
          "engine_name": "Microsoft",
          "engine_version": "1.1.20200.4",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "SUPERAntiSpyware": {
          "category": "undetected",
          "engine_name": "SUPERAntiSpyware",
          "engine_version": "5.6.0.1032",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230409"
        },
        "ZoneAlarm": {
          "category": "undetected",
          "engine_name": "ZoneAlarm",
          "engine_version": "1.0",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "GData": {
          "category": "undetected",
          "engine_name": "GData",
          "engine_version": "A:25.35610B:27.31281",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "Google": {
          "category": "undetected",
          "engine_name": "Google",
          "engine_version": "1681131643",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "BitDefenderFalx": {
          "category": "type-unsupported",
          "engine_name": "BitDefenderFalx",
          "engine_version": "2.0.936",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230203"
        },
        "AhnLab-V3": {
          "category": "undetected",
          "engine_name": "AhnLab-V3",
          "engine_version": "3.23.2.10388",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "Acronis": {
          "category": "undetected",
          "engine_name": "Acronis",
          "engine_version": "1.2.0.114",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230219"
        },
        "McAfee": {
          "category": "undetected",
          "engine_name": "McAfee",
          "engine_version": "6.0.6.653",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "MAX": {
          "category": "undetected",
          "engine_name": "MAX",
          "engine_version": "2023.1.4.1",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "DeepInstinct": {
          "category": "type-unsupported",
          "engine_name": "DeepInstinct",
          "engine_version": "3.1.0.15",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230409"
        },
        "VBA32": {
          "category": "undetected",
          "engine_name": "VBA32",
          "engine_version": "5.0.0",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "Cylance": {
          "category": "type-unsupported",
          "engine_name": "Cylance",
          "engine_version": "2.0.0.0",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230406"
        },
        "Zoner": {
          "category": "undetected",
          "engine_name": "Zoner",
          "engine_version": "2.2.2.0",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230409"
        },
        "Tencent": {
          "category": "undetected",
          "engine_name": "Tencent",
          "engine_version": "1.0.0.1",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "Yandex": {
          "category": "timeout",
          "engine_name": "Yandex",
          "engine_version": "5.5.2.24",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "TACHYON": {
          "category": "undetected",
          "engine_name": "TACHYON",
          "engine_version": "2023-04-10.02",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "MaxSecure": {
          "category": "undetected",
          "engine_name": "MaxSecure",
          "engine_version": "1.0.0.1",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230408"
        },
        "Fortinet": {
          "category": "undetected",
          "engine_name": "Fortinet",
          "engine_version": "6.4.258.0",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "BitDefenderTheta": {
          "category": "undetected",
          "engine_name": "BitDefenderTheta",
          "engine_version": "7.2.37796.0",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230403"
        },
        "AVG": {
          "category": "undetected",
          "engine_name": "AVG",
          "engine_version": "22.11.7701.0",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "Panda": {
          "category": "undetected",
          "engine_name": "Panda",
          "engine_version": "4.6.4.2",
          "result": null,
          "method": "blacklist",
          "engine_update": "20230410"
        },
        "CrowdStrike": {
          "category": "type-unsupported",
          "engine_name": "CrowdStrike",
          "engine_version": "1.0",
          "result": null,
          "method": "blacklist",
          "engine_update": null
        }
      },
      "trid": [
        {
          "file_type": "7-Zip compressed archive (v0.4)",
          "probability": 57.1
        },
        {
          "file_type": "7-Zip compressed archive (gen)",
          "probability": 42.8
        }
      ],
      "sha256": "ec18d9557ce383d0d8bf04d162bea2c289bd1ae6cdc4985e9537d7eb01b6d7b4",
      "tags": [
        "7zip"
      ],
      "last_analysis_date": 1681133570,
      "unique_sources": 67,
      "first_submission_date": 1680619453,
      "ssdeep": "1572864:dQEzyldIj2H0w6JWucGRkAotiQve1rY7Xii169OXsft97G+vHgKuUXFDawqbGrku:mEOdIjgIRkAWveNiojVvH7a2wOH5",
      "md5": "11ffacd028746c4d00aca32bfd6a9f4c",
      "sha1": "ad08eedcef43d8d4600ec02e3c789b8cbdcc5f7c",
      "magic": "7-zip archive data, version 0.4",
      "last_analysis_stats": {
        "harmless": 0,
        "type-unsupported": 16,
        "suspicious": 0,
        "confirmed-timeout": 0,
        "timeout": 5,
        "failure": 0,
        "malicious": 0,
        "undetected": 53
      },
      "meaningful_name": "LosslessCut 3.54.0 #soft8 病毒 0 (112.04.11).7z",
      "reputation": 0
    },
    "type": "file",
    "id": "ec18d9557ce383d0d8bf04d162bea2c289bd1ae6cdc4985e9537d7eb01b6d7b4",
    "links": {
      "self": "https://www.virustotal.com/api/v3/files/ec18d9557ce383d0d8bf04d162bea2c289bd1ae6cdc4985e9537d7eb01b6d7b4"
    }
  }
}
```
  
  ```javascript
  console.log("I'm a code block!");
  ```
  
</details>

So 74 of the 89 scanners are for AV. For this file, 53 passed it, 16 were scanners for other types of files, and 5 scanners timed out.

**Reply by mifi** [reply]
Thanks for checking. Sounds like a false positive then? Besides i never created any releases with all these file names, so maybe someone made a malware copy of losslesscut and distributed it, confusing virustotal’s algorithm:

        "LosslessCut 3.54.0 #soft8 病毒 0 (112.04.11).7z",
        "LosslessCut-win-x64.7z",
        "losslesscut-win-x64.7z",
        "LosslessCut-win-x64 (1).7z",
        "LosslessCut-win-x64_2.7z",
        "LosslessCut-win-x64(1).7z",
        "LosslessCut-win-x64 (4).7z",
        "LosslessCut 3.54.0 Standantalone_x64.7z",
        "LosslessCut-win-x64 3.54.7z",
        "LosslessCut v3.54.0 Fr Portable.7z",
        "LosslessCut-win-x64-3.54.0.7z",
        "LosslessCut-win-x64 (April 4, 2023).7z"

**Reply by photonometric** [reply]
> Sounds like a false positive then? 

Yes it is definitely safe to say this—but more accurately, it was a **false positive of the GitHub website itself**; the result would have been the same with any file hosted on github, or indeed any regular page on the GitHub site. LosslessCut was essentially an "innocent bystander" for this URL scan: the 1 positive result was from one of VirusTotal's **URL/domain scanners** (Lumu), which are completely different from the antivirus/malware file scanning software like what we're used to on our desktops. 

While one of VT's AV/malware scanners would indeed have indicated a (false) positive of the [LosslessCut release file](https://github.com/mifi/lossless-cut/releases/download/v3.54.0/LosslessCut-win-x64.7z), the URL/domain scanners are instead concerned with the authenticity of the web server itself. These scan for things like takeover/redirects by phishing sites, which mimic e.g. Paypal login pages to harvest user login info; or servers which are potentially hacked/botnet/cyberattack culprits, etc. 

Needless to say, this did not happen to GitHub; but just like a file scan, these URL scans use behavioral characteristics to predict what _could_ be a danger sign (and often get it wrong). This might have been a GitHub server outage, SSL cert error, server software upgrade, or even an inaccurate clock on the GitHub site. 

> Besides i never created any releases with all these file names, so maybe someone made a malware copy of losslesscut and distributed it, confusing virustotal’s algorithm:

Actually VT scanned only one file, and it was indeed the legit file from your release link. These other filenames are from users who downloaded the release, renaming it various things as they saved it to their computers. Then users uploaded these files to the VT website, or were simply auto-scanned when downloaded, by users who installed VT's browser plugins (which scan every file they download). 

These variously-named 7z archives were all the same file, with the same MD5 hash, so VT was simply listing these as other identical "variations" of the file from this particular URL file scan. Any alteration to the file would have altered the MD5, and would be listed as a separate report like all other files. There is no question of any of these containing any malware either; the VT database operates by file hash, and provides this same report even if the file is uploaded 1000 different times with completely different names. 

Although if you upload LosslessCut-win-x64.7z on the VT scanning website, it will return NO positives. This is because a simple AV/malware file scan has no URL component (like the github release link) to submit to engines like Lumu, so it would not include results from the 15 URL scanners. While a VT scan of the GitHub url containing this identical file does return domain scanner results (including the Lumu false positive), which have literally nothing to do with the contents of the file, but only analyze info about the server where it downloads that file from (github).

Sorry for the long and somewhat redundant explanation. VirusTotal is a bit confusing (and isn't helped by being a mysterious "black box" that doesn't explain its results or how the scanner software determines a "positive"). And this kind of alert is concerning enough that it's worth explaining fully how this was completely erroneous, so people don't have nagging suspicions.

**Reply by photonometric** [reply]
@mifi Also, it might be worth adding a [false positive] tag to the title of this discussion thread, so no one is scared off by false info if they don't take the time to read the whole thread ;)

**Reply by mifi** [reply]
thanks for the analysis👍
